website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Data Lake Analytics: Alibaba's Federated Cloud Strategy
+author: George Wang
+authorURL: https://www.linkedin.com/in/george-wang-9a5a46190/
+---
+
+
+## Data Federation Service
+
+Today’s cloud database empowers cloud users to easily and directly interact with other cloud products and services, available to be built for their own cloud data lake analytic scenarios and solutions. The architecture to pile up a cloud-based data lake analytic scenario is divided into three layers:
```

Comment:
Can you refactor this enough that there is an index and then add the truncate tag (you can see it in other blog posts).

This will prevent the entire blog post from appearing in the index. It will only show everything above the truncate tag.

Reply:
got it, I have revised by adding an short intro with index paragraph